src/js/popup/popup.js

```diff
@@ -269,13 +298,10 @@ async function showRelayPanel(tipPanelToShow) {
   const isBundleAvailableInCountry = (await browser.storage.local.get("bundlePlans")).bundlePlans.BUNDLE_PLANS.available_in_country;
   const isPhoneAvailableInCountry = (await browser.storage.local.get("phonePlans")).phonePlans.PHONE_PLANS.available_in_country;
 
-  const hasPhone = (await browser.storage.local.get("has_phone")).has_phone;
-  const hasVpn = (await browser.storage.local.get("has_vpn")).has_vpn;
-
   // Conditions for phone masking announcement to be shown: if the user is in US/CAN, phone flag is on, and user has not purchased phone plan yet
-  const isPhoneMaskingAvailable = isPhoneAvailableInCountry && !hasPhone;
+  const isPhoneMaskingAvailable = await checkWaffleFlag("phones") && isPhoneAvailableInCountry;
```

Comment:
@Vinnl Is this a regression?

Reply:
@maxxcrawford I think I'm missing some context here, as I don't quite understand the question :/